ts/nni_manager/training_service/local/localTrainingService.ts

```diff
@@ -414,13 +414,13 @@ class LocalTrainingService implements TrainingService {
         if (process.platform === 'win32') {
             script.push(`cd $env:NNI_CODE_DIR`);
             script.push(
-                `cmd.exe /c ${this.config.trialCommand} 2>&1 | Out-File "${path.join(workingDirectory, 'stderr')}" -encoding utf8`,
+                `cmd.exe /c ${this.config.trialCommand} 1> "${path.join(workingDirectory, 'stdout')}" 2> "${path.join(workingDirectory, 'stderr')}" -encoding utf8`,
                 `$NOW_DATE = [int64](([datetime]::UtcNow)-(get-date "1/1/1970")).TotalSeconds`,
                 `$NOW_DATE = "$NOW_DATE" + (Get-Date -Format fff).ToString()`,
                 `Write $LASTEXITCODE " " $NOW_DATE  | Out-File "${path.join(workingDirectory, '.nni', 'state')}" -NoNewline -encoding utf8`);
         } else {
             script.push(`cd $NNI_CODE_DIR`);
-            script.push(`eval ${this.config.trialCommand} 2>"${path.join(workingDirectory, 'stderr')}"`);
+            script.push(`eval ${this.config.trialCommand} 1>"${path.join(workingDirectory, 'stdout')} 2>"${path.join(workingDirectory, 'stderr')}"`);
```

Comment:
We still cannot view stdout from WebUI. That should be a problem?

Reply:
FIxed,

After some investigate about this issue, here are some result:
_option1:_
sys.stdout redirect for trial.py (->trial.log)
_result:_
only trial.py log show up, no subprocess log. (This is the issue linked on this PR)

_option2:_
sys.stdout redirect for trial.py (->trial.log)
launch trial.py with command "1 > stdout"
_result:_
trial.py log is in trial.log, subprocess log is in stdout.

_option3:_
sys.stdout redirect for trial.py (->trial.log)
launch trial.py with command "1 > trial.log"
_result:_
Thread save exceptions.

_option4:_
launch trial.py with command "1 > stdout"
_result:_
trial.py log and subprocess log are in stdout.

I recommend remove sys.stdout redirect which is only works for local platform.

_(Somehow, It is no works for subprocess output if only setup "sys.stdout" on parent process, You have to change the code by "subprocess.Popen(your_command, stdout=sys.stdout)")_